x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -258,10 +256,9 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     cy.url().should('include', expectedVal);
   });
 
-  it(`Filters are restored from localstorage when user navigates back to the page.`, () => {
+  it(`should restore Filters from localstorage when user navigates back to the page.`, () => {
     cy.get(OPTION_LIST_VALUES(1)).click();
```

Comment:
should we extract this into a `task` function?

Reply:
@PhilippeOberti , Sorry I do not 100% understand what you think could be extracted to a `task`. Is it going away navigating back to the original page? 

I believe we could but is it very generic that it warrants a separate function?

Reply:
I was told that every click event should be extracted to a task. While I agree with you that sometimes (most of the time actually) I would prefer having those stay in the test directly, I've religiously obeyed and extracted all of these into tasks... that's why I'm mentioning it here, just for consistency...

Reply:
```
cy.get(OPTION_LIST_VALUES(1)).click();
cy.get(OPTION_SELECTABLE(1, 'high')).click();
``` 
would be extracted out under a tasks function

Reply:
Actually, below step is already a task called `openFilterPopover` and I have replaced it.

```typescript
cy.get(OPTION_LIST_VALUES(1)).click();
```
And when that is executed, it is not necessary that will perform the click action. So I would rather not club them.